src/app/core/messages.service.ts

```diff
@@ -9,7 +9,7 @@ export class Message {
 }
 @Injectable()
 export class MessagesService {
-  private _message: Message;
+  private _message: Message = new Message(true, 'Clark is now accepting curriculum submissions to the Plan C collection.');
```

Comment:
We can make Plan C a link to the collection page.

Reply:
How to I add a link?  I tried doing embedded <a> tag but it didn't convert to a link.